catroid/src/org/catrobat/catroid/common/Constants.java

```diff
@@ -109,6 +109,11 @@
 	public static final int DOWNLOAD_NOTIFICATION = 103;
 	public static final int COPY_NOTIFICATION = 104;
 
+	//flavour (Pocket Code, Pocket Phiro,.....)
```

Comment:
Nitpick: *flavor*. Also, do we even need this comment if it doesn't really explain what the flavors do or what they are for? Currently it just lists the constants below.